### ISSUE TITLE
Create loopback4 community organization

### ISSUE THREAD

**Issue by mrbatista** [strongloop/loopback-next#6203]
## Suggestion
Hi, like [ionic capacitor community](https://github.com/capacitor-community) what do you think if we created a loopback4 community organization that contains plugins, tools, and other projects that are driven by the community?

**Comment by dhmlau** [comment]
@mrbatista, thanks for your suggestion. There is a https://github.com/strongloop-community, would that be something you're looking for?

**Comment by mrbatista** [comment]
@dhmlau Yes! How user can candidate own project to be listed here in or create new one?

**Comment by dhmlau** [comment]
That's a good question. I think you need to be a member of that github org at the minimum (I can add you there). 
I'm not sure how the projects were added in the past, because looking at the settings, it doesn't seem like members can create a repo (though they have other rights). Maybe @bajtos might know.

**Comment by bajtos** [comment]
Thank you @mrbatista for starting the discussion. Let me move this issue to [loopback-governance](https://github.com/strongloop/loopback-governance) to make it stand out from regular project "noise".
